src/PaymentMethodCollectElement.res

```diff
@@ -16,6 +17,7 @@ let make = (~integrateError, ~logger) => {
   )
   let (amount, setAmount) = React.useState(_ => options.amount)
   let (currency, setCurrency) = React.useState(_ => options.currency)
+  let (formLayout, setFormLayout) = React.useState(_ => options.formLayout)
```

Comment:
Why are u creating another state for formLayout? You can directly use options.formLayout wherever required